.github/workflows/pipelines-root.yml

```diff
@@ -53,36 +53,47 @@ jobs:
     name: Detect Infrastructure Changes
     runs-on: ${{ fromJSON(inputs.runner) }}
     steps:
+      - name: Sanitize Tokens
+        id: secrets
+        shell: bash
+        run: |
+          PR_TRIM=$(echo $PIPELINES_READ_TOKEN | xargs)
+          IR_TRIM=$(echo $INFRA_ROOT_WRITE_TOKEN | xargs)
+          OR_TRIM=$(echo $ORG_REPO_ADMIN_TOKEN | xargs)
+          echo "PIPELINES_READ_TOKEN=$PR_TRIM" | xargs>> $GITHUB_OUTPUT
```

Comment:
What happened here? Is this intentional?

Reply:
nope, thanks for finding!  Shouldn't be repeating the xargs